universal-login-react/src/ui/Notifications/EmojiPlaceholders.tsx

```diff
@@ -14,7 +14,7 @@ export const EmojiPlaceholders = ({code, onEmojiClicked, maxLength}: EmojiPlaceh
     const emojis = [];
 
     for (let i = 0; i < maxLength; i++) {
-      if (code[i]) {
+      if (typeof code[i] === 'number') {
```

Comment:
Ugly. Can we give it a type? Why would it not be a number?

Reply:
1. We use **i** for indexing React components. 
2. `codes.length` can have `length !==0 && length !== maxLength`. This component use  placeholders for button which appears when you click some code in FakeCodes. 
3. code[i] can be falsy value (`code[i]=0 || undefined`) that's why wee need check type or `code[i] !== undefined` to know what should be render `<li><button /></li>` or just `<li />`